src/lib.rs

```diff
@@ -481,16 +487,21 @@ impl LineParser {
 ///
 /// For basic usage, see the crate-level documentation.
 /// Note that once `next` returns an error, the result of further calls is undefined.
-pub struct PropertiesIter<R: Read> {
-  lines: LogicalLines<NaturalLines<R>>,
+pub struct PropertiesIter<'a, R: Read> {
```

Comment:
... I got stuck on `Encoding`'s lack of size, and didn't really want to see `Box<Encoding>` in the API, anyway, so I took your suggestion of `&'static Encoding`.